active/ZEP0000.md

```diff
@@ -60,29 +66,41 @@ that require collaboration across multiple projects.
 
 - **Specification ZEP**
 
-Specification ZEPs deal with the changes related to [Zarr Specifications].
-The SPEC ZEP (abbreviation of Specification ZEP) for now introduces changes in
-the core specification of Zarr. The changes related to core specification follow a
+Specification ZEPs deal with the changes related to [Zarr Specifications]. The
+SPEC ZEP (abbreviation of Specification ZEP) introduces changes to the
+specification of Zarr. The changes related to specification follow a
 strict and thorough review process and should be adopted by everyone in the
-Zarr community.
+Zarr community. Specification ZEPs are of two types. They are:
 
 - **Core protocol ZEP**
 
+<!-- Mention Zarr-Spec devs in line: 85-87, after zarr-developers/governance/#44 is finalised -->
+
 Describes a ZEP which involves changes in the core specification of Zarr. Core
-protocol ZEPs (commonly known as Core ZEPs) are a part of SPEC ZEPs and apply
-to [Zarr Specifications] and its various implementations under the
-[zarr-developers/zarr_implementations] GitHub repo. The core protocol ZEPs
-should be adopted by every implementation of Zarr and, in general, the overall
-Zarr community. Core ZEPs must go through a thorough review process, including
-involvement, discussion, and voting from the author of the proposed ZEP, Zarr
-Steering Council, author(s) of various Zarr implementations (ZIC), and
-open-source projects/research groups using Zarr and the general Zarr community.
-The general advice is that everyone should be made aware of changes introduced
-by core ZEPs. Core protocol ZEPs require community consensus, and developers or
-users are typically not free to ignore them.
-
-Note: Currently, Specification ZEPs only deal with the core specification of Zarr.
-The scope of SPEC ZEPs will be extended in the future.
+protocol ZEPs (hereafter referred to as "Core ZEPs") are a type of Spec ZEPs,
+and apply to [Zarr Specifications] and its various implementations under the
+[zarr.dev/implementations]. The core protocol ZEPs should be adopted by every
+implementation of Zarr and, in general, the overall Zarr community. Core ZEPs
+must go through a thorough review process, including involvement, discussion,
+and voting from the author of the proposed ZEP, Zarr Steering Council, author
+(s) of various Zarr implementations (ZIC), and open-source projects/research
+groups using Zarr and the general Zarr community. The general advice is that
+everyone should be made aware of changes introduced by core ZEPs. Core protocol
+ZEPs require community consensus, and developers or users are typically not
+free to ignore them. 
```

Comment:
Given the fact that we have decided not to version the V3 spec (see https://github.com/zarr-developers/zarr-specs/issues/277), it would be helpful to state explicitly whether any core protocol ZEP (i.e. "changes in the core specification of Zarr") will generate a new major version of the spec.

In my opinion, these should go together. Any change major enough to warrant a core protocol ZEP would also trigger a major version increment. The next core protocol ZEP will produce Zarr V4.

Reply:
Done. Added via https://github.com/zarr-developers/zeps/pull/59/commits/3ae1fc5833cc95220e87370c308b20619f4ab931.